docs/docs-ref-autogen/excel/excelscript/excelscript.rangecopytype.yml

```
@@ -4,7 +4,35 @@ uid: 'ExcelScript!ExcelScript.RangeCopyType:enum'
 package: ExcelScript!
 fullName: ExcelScript.RangeCopyType
 summary: ''
-remarks: ''
+remarks: |-
+
+
+  #### Examples
+
+  ```TypeScript
+  /**
+   * This script copies the values from the current worksheet to a new worksheet.
```

A minor addition, because I wondered whether the sample copies all values or only some values while reading the description. 
```suggestion
   * This script copies all of the values from the current worksheet to a new worksheet.
```